### PR TITLE
Fix i18n for bindings

### PR DESCRIPTION
**Pull request by J-N-K**
Regression of #13289 - see https://github.com/openhab/openhab-addons/pull/13289#issuecomment-1384511682

Signed-off-by: Jan N. Klug <github@klug.nrw>

### CODE REVIEW

**Comment by jlaur** [comment]
@J-N-K - minor, but do you think you can also replace the header `# binding` with `# addon` now that you're at it? This is also how it's generated by `i18n:generate-default-translations`.

File: bundles/org.openhab.binding.allplay/src/main/resources/OH-INF/i18n/allplay.properties
Comment (jlaur) [inline bundles/org.openhab.binding.allplay/src/main/resources/OH-INF/i18n/allplay.properties:8]: Is this correct?

**Comment by jlaur** [comment]
@openhab/add-ons-maintainers. please do not merge yet. I think too much has been replaced, so probably need one more iteration.

**Comment by Hilbrand** [comment]
Shouldn't this only be changed for the `en` version. For all other languages this must be changed in crowdin?

**Comment by jlaur** [comment]
> Shouldn't this only be changed for the `en` version. For all other languages this must be changed in crowdin?

Theoretically, yes. I'm not sure though if Crowdin will actually push back **all** translations after a merge? If anything is missed, those translations will be missing for the users. There should be no conflicts, so maybe it would be safer to apply directly? Cc @lolodomo

File: bundles/org.openhab.binding.allplay/src/main/resources/OH-INF/i18n/allplay.properties
Comment (J-N-K) [reply bundles/org.openhab.binding.allplay/src/main/resources/OH-INF/i18n/allplay.properties:8]: IMO yes, but I‘m no expert. It‘s configuration at add-on level.

Comment (jlaur) [reply bundles/org.openhab.binding.allplay/src/main/resources/OH-INF/i18n/allplay.properties:8]: I'll compare with i18n tool output later.

File: bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties
Comment (andrewfg) [inline bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties:14]: I think there is potentially an issue with this binding information thing. Probably the thing.xml needs to be changed as well?

File: bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties
Comment (andrewfg) [inline bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties:34]: Same as above

**Review by andrewfg — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.zway/src/main/resources/OH-INF/i18n/zway.properties
Comment (andrewfg) [inline bundles/org.openhab.binding.zway/src/main/resources/OH-INF/i18n/zway.properties:11]: Possibly same issue as in my comment on Velux ??

**Review by andrewfg — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties
Comment (J-N-K) [reply bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties:14]: No, that one should not have been amended. `binding` is the thing name in this case and that's fine (besides that IMO a thing that modifies the binding behavior should never have made it into the code-base, it's totally anti-pattern).

File: bundles/org.openhab.binding.zway/src/main/resources/OH-INF/i18n/zway.properties
Comment (J-N-K) [reply bundles/org.openhab.binding.zway/src/main/resources/OH-INF/i18n/zway.properties:11]: See above.

File: bundles/org.openhab.binding.heliosventilation/src/main/resources/OH-INF/i18n/heliosventilation_de.properties
Comment (andrewfg) [inline bundles/org.openhab.binding.heliosventilation/src/main/resources/OH-INF/i18n/heliosventilation_de.properties:3]: You probably need to look at this one? I have a feeling that there should not be an 'iCloud' prefix on these lines. Or??

**Review by andrewfg — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.icloud/src/main/resources/OH-INF/i18n/iCloud.properties
Comment (andrewfg) [inline bundles/org.openhab.binding.icloud/src/main/resources/OH-INF/i18n/iCloud.properties:2]: You probably need to look at this one? I have a feeling that there should not be an 'iCloud' prefix on these lines. Or??

**Review by andrewfg — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties
Comment (andrewfg) [reply bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties:14]: > a thing that modifies the binding behavior should never have made it into the code-base

Agreed. (Not my code). But to be fair, it does not modify any behavior; it simply provides some static text information; but if I would write the binding myself, that information could be in a bridge property, rather than a biding thing..

**Review by stefan-hoehn — APPROVED** [review]
LGTM: Nanoleaf

File: bundles/org.openhab.binding.allplay/src/main/resources/OH-INF/i18n/allplay.properties
Comment (jlaur) [reply bundles/org.openhab.binding.allplay/src/main/resources/OH-INF/i18n/allplay.properties:8]: You are right, same is generated by `i18n:generate-default-translations`.

File: bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/i18n/androiddebugbridge_it.properties
Comment (jlaur) [inline bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/i18n/androiddebugbridge_it.properties:29]: This should be have been replaced.
```suggestion
thing-type.config.androiddebugbridge.android.mediaStateJSONConfig.description = Configurazione JSON che permette di modificare la strategia di rilevamento dello stato dei media per ogni app. Fare riferimento alla documentazione del binding.
```

File: bundles/org.openhab.binding.dsmr/src/main/resources/OH-INF/i18n/dsmr_nl.properties
Comment (jlaur) [inline bundles/org.openhab.binding.dsmr/src/main/resources/OH-INF/i18n/dsmr_nl.properties:26]:
```suggestion
error.connector.internal_error = Onverwachte fout, mogelijk een fout in de binding. Maak hier een melding van.   
```

File: bundles/org.openhab.binding.http/src/test/resources/simplelogger.properties
Comment (jlaur) [inline bundles/org.openhab.binding.http/src/test/resources/simplelogger.properties:2]: Not sure about this one.

File: bundles/org.openhab.binding.icloud/src/main/resources/OH-INF/i18n/iCloud.properties
Comment (jlaur) [reply bundles/org.openhab.binding.icloud/src/main/resources/OH-INF/i18n/iCloud.properties:2]: In addon.xml:
```xml
	<name>@text/icloud.binding.name</name>
	<description>@text/icloud.binding.description</description>
```

So good catch, it would probably be better to fix addon.xml to refer to icloud.addon.*

File: bundles/org.openhab.binding.kvv/src/main/resources/OH-INF/i18n/kvv.properties
Comment (jlaur) [inline bundles/org.openhab.binding.kvv/src/main/resources/OH-INF/i18n/kvv.properties:11]:
```suggestion
thing-type.kvv.stop.description = Train stop for KVV Binding.
```

File: bundles/org.openhab.binding.lgwebos/src/test/resources/simplelogger.properties
Comment (jlaur) [inline bundles/org.openhab.binding.lgwebos/src/test/resources/simplelogger.properties:2]: Might also need to be checked.

File: bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/i18n/mielecloud.properties
Comment (jlaur) [inline bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/i18n/mielecloud.properties:3]:
```suggestion
addon.mielecloud.description=This is the cloud-based Miele@home binding.
```

Comment (J-N-K) [reply bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/i18n/mielecloud.properties:3]: I don't think this should be changed. It's still a binding. The whole idea of the core change is to extend the metadata to the other add-on types besides a binding, not to replace "binding" by "add-on".

File: bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/i18n/mielecloud_de.properties
Comment (jlaur) [inline bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/i18n/mielecloud_de.properties:3]:
```suggestion
addon.mielecloud.description=Dies ist die Cloud-basierte Miele@home-Binding.
```

File: bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/i18n/mielecloud.properties
Comment (jlaur) [reply bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/i18n/mielecloud.properties:3]: That's why I suggest to change it back. 🙂

**Review by pacive — APPROVED** [review]

File: bundles/org.openhab.binding.http/src/test/resources/simplelogger.properties
Comment (J-N-K) [reply bundles/org.openhab.binding.http/src/test/resources/simplelogger.properties:2]: This should not have been changed, correct. But why is this file there?

**Review by Wolfgang1966 — APPROVED** [review]
LGTM for awattar and tacmi. Thank you

File: bundles/org.openhab.binding.icloud/src/main/resources/OH-INF/i18n/iCloud.properties
Comment (J-N-K) [reply bundles/org.openhab.binding.icloud/src/main/resources/OH-INF/i18n/iCloud.properties:2]: Why is that defined in this way at all? My understanding was that this is replaced automatically.

**Comment by jlaur** [comment]
> I would say the most important thing is to figure out how we can rename the already existing translations on CrowdIn, so the changed versions are not overwritten.

True. However, nothing will come back from Crowdin unless 100% of the strings are translated (and approved). So nothing should be overwritten unless someone would manually overwrite in Crowdin.

File: bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties
Comment (andrewfg) [inline bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties:15]: And this.

File: bundles/org.openhab.binding.regoheatpump/src/main/resources/OH-INF/i18n/regoheatpump.properties
Comment (andrewfg) [inline bundles/org.openhab.binding.regoheatpump/src/main/resources/OH-INF/i18n/regoheatpump.properties:4]: And this (last 'addon')

File: bundles/org.openhab.binding.network/src/main/resources/OH-INF/i18n/network.properties
Comment (andrewfg) [inline bundles/org.openhab.binding.network/src/main/resources/OH-INF/i18n/network.properties:57]: And this.

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties
Comment (andrewfg) [inline bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties:97]: Probably this.

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties
Comment (andrewfg) [inline bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties:133]: And this.

File: bundles/org.openhab.binding.kaleidescape/src/main/resources/OH-INF/i18n/kaleidescape.properties
Comment (andrewfg) [inline bundles/org.openhab.binding.kaleidescape/src/main/resources/OH-INF/i18n/kaleidescape.properties:54]: Another one :)

**Review by andrewfg — CHANGES_REQUESTED** [review]
A few more potential issues..

File: bundles/org.openhab.io.neeo/src/main/resources/OH-INF/i18n/neeo_de.properties
Comment (jlaur) [inline bundles/org.openhab.io.neeo/src/main/resources/OH-INF/i18n/neeo_de.properties:6]:
```suggestion
io.config.neeo.exposeNeeoBinding.label = NEEO-Bindung aufdecken
io.config.neeo.exposeNeeoBinding.description = Things aufdecken, die durch die NEEO-Bindung gefunden wurden
```

File: bundles/org.openhab.io.neeo/src/main/resources/OH-INF/i18n/neeo.properties
Comment (jlaur) [inline bundles/org.openhab.io.neeo/src/main/resources/OH-INF/i18n/neeo.properties:6]:
```suggestion
io.config.neeo.exposeNeeoBinding.label = Expose NEEO Binding
io.config.neeo.exposeNeeoBinding.description = Expose things found by the NEEO Binding
```

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max_de.properties
Comment (jlaur) [inline bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max_de.properties:98]:
```suggestion
thing-type.config.max.thermostat.group.addon.description = Einstellungen für das Binding.
```

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max_de.properties
Comment (jlaur) [inline bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max_de.properties:134]:
```suggestion
thing-type.config.max.thermostatplus.group.addon.description = Einstellungen für das Binding.
```

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max_de.properties
Comment (jlaur) [inline bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max_de.properties:133]:
```suggestion
thing-type.config.max.thermostatplus.group.binding.label = Binding
```

**Review by bern77 — APPROVED** [review]

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max_de.properties
Comment (jlaur) [inline bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max_de.properties:97]:
```suggestion
thing-type.config.max.thermostat.group.binding.label = Binding
```

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties
Comment (jlaur) [inline bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties:97]:
```suggestion
thing-type.config.max.thermostat.group.binding.label = Binding Settings
```

This is from:

```xml
			<parameter-group name="binding">
				<label>Binding Settings</label>
				<description>Binding settings for this device</description>
			</parameter-group>
```

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties
Comment (jlaur) [inline bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties:98]:
```suggestion
thing-type.config.max.thermostat.group.binding.description = Binding settings for this device
```

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties
Comment (jlaur) [inline bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties:134]:
```suggestion
thing-type.config.max.thermostatplus.group.binding.description = Binding settings for this device
```

File: bundles/org.openhab.binding.icloud/src/main/resources/OH-INF/i18n/iCloud.properties
Comment (jlaur) [reply bundles/org.openhab.binding.icloud/src/main/resources/OH-INF/i18n/iCloud.properties:2]: My guess would be to avoid redundancy, but it seems strange to do this when we have the i18m tool.

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max_de.properties
Comment (jlaur) [reply bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max_de.properties:98]: @J-N-K - I went through all files now. Sorry for the messy review. I believe this is my last finding.

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties
Comment (jlaur) [reply bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties:133]: Not sure what happened here, but still wrong.
```suggestion
thing-type.config.max.thermostatplus.binding.addon.label = Binding Settings
```

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties
Comment (jlaur) [reply bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties:97]: Also still wrong.

File: bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties
Comment (jlaur) [reply bundles/org.openhab.binding.max/src/main/resources/OH-INF/i18n/max.properties:134]: And here.

**Review by janvyb — APPROVED** [review]

**Review by jlaur — APPROVED** [review]
Thanks! @openhab/add-ons-maintainers, I went through all files and they look good to me now. I  also committed to the branch, so will need another pair of eyes.

**Review by EvilPingu — APPROVED** [review]

**Comment by jlaur** [comment]
@J-N-K - if you can approve/"take ownership" of my two commits, I should be able to merge this in accordance with our policies. Waiting any longer will just increase the risk of conflicts from Crowdin, and this PR is quite heavy to dance with.

**Comment by jlaur** [comment]
@cweitkamp - if you are still active today, can you merge this one as well?

**Review by mvalla — APPROVED** [review]

**Comment by mvalla** [comment]
LGTM for openwebnet

**Comment by lolodomo** [comment]
Note that entry "addon.gree.label" has to be removed, I will create a PR.

Is it normal to not have any updated file for io / persistence / transform / voice bindings ?

**Comment by lolodomo** [comment]
> Theoretically, yes. I'm not sure though if Crowdin will actually push back **all** translations after a merge? If anything is missed, those translations will be missing for the users. There should be no conflicts, so maybe it would be safer to apply directly? Cc @lolodomo

I am now expecting we will have to fill again all translated binding name and description in Crowdin !
The PR is merged, so this can be verified now.

**Comment by hmerk** [comment]
@J-N-K Don't know if this is is related to this PR, but I now see two incorrect names for installed Bindings with SNAPSHOT 3288

image

**Comment by jlaur** [comment]
@hmerk - yes, related: https://github.com/openhab/openhab-addons/pull/14235/#discussion_r1072164659

**Comment by jlaur** [comment]
I will create a small PR for fixing that as well as one other issue mentioned by @lolodomo: https://github.com/openhab/openhab-addons/pull/13289#issuecomment-1399231285

**Comment by lolodomo** [comment]
iCloud and Shelly look good to me with snapshot 3286.
image
image

**Comment by hmerk** [comment]
@lolodomo As I wrote, screenshots are from SNAPSHOT 3288, created today 😉

**Comment by jlaur** [comment]
@openhab/add-ons-maintainers - for some reason some strings were not imported into Crowdin. For example, see the Miele@home binding:
```
# add-on

addon.miele.name = Miele Binding
addon.miele.description = This is the binding for Miele@home appliances
```

Now see [Crowdin](https://crowdin.com/translate/openhab-addons/1675/en-da?filter=basic&value=0):

image

**Pull request merged**
